README.md

```diff
@@ -129,14 +129,14 @@ class YourFavoriteModel(BaseEstimator): # Inherits sklearn base classifier
         pass
 
 # Now you can use your model with `cleanlab`. Here's one example:
-from cleanlab.classification import LearningWithNoisyLabels
-lnl = LearningWithNoisyLabels(clf=YourFavoriteModel())
-lnl.fit(train_data, train_labels_with_errors)
+from cleanlab.classification import CleanLearning
+cl = CleanLearning(clf=YourFavoriteModel())
+cl.fit(train_data, train_labels_with_errors)
```

Comment:
Maybe add comment that: all of the methods in `clf` (such as `predict()`) will be available in `cl` as well.

Reply:
👍